sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2SQLSuite.scala

```diff
@@ -2476,22 +2456,6 @@ class DataSourceV2SQLSuite
       .head().getString(1) === expectedComment)
   }
 
-  test("SPARK-30799: temp view name can't contain catalog name") {
-    val sessionCatalogName = CatalogManager.SESSION_CATALOG_NAME
-    withTempView("v") {
-      spark.range(10).createTempView("v")
-      val e1 = intercept[AnalysisException](
-        sql(s"CACHE TABLE $sessionCatalogName.v")
-      )
-      assert(e1.message.contains(
-        "Table or view not found: spark_catalog.v"))
-    }
-    val e2 = intercept[AnalysisException] {
-      sql(s"CREATE TEMP VIEW $sessionCatalogName.v AS SELECT 1")
```

Comment:
It's a parser error, I moved the test to https://github.com/apache/spark/pull/34490/files#diff-0db82228a944d6070203b6ac834c3c8d9b16b985fc3ff319ed7408fc3b6a6c23R363